### PR TITLE
enhancement: Prod 2310/backfill chart versions on compatibility tables

### PR DESCRIPTION
**Pull request by seemywingz**
Updated the `ingress-nginx` scraper to support new readme for nginx and k8s-supported versions
Add helm chart version updater

## Test Plan

I tested manually and verified the chart versions match the app versions in both the original hem index.yaml and the updated compatibility tables

## Checklist

- [X] If required, I have updated the Plural documentation accordingly.
- [ ] I have added tests to cover my changes.
- [X] I have added a meaningful title and summary to convey the impact of this PR to a user.

### CODE REVIEW

File: static/compatibilities/cert-manager.yaml [hunk 1]
```diff
@@ -1,99 +1,122 @@
 icon: https://raw.githubusercontent.com/cert-manager/cert-manager/d53c0b9270f8cd90d908460d69502694e1838f5f/logo/logo-small.png
 git_url: https://github.com/cert-manager/cert-manager
 release_url: https://github.com/cert-manager/cert-manager/releases/tag/v{vsn}
+chart_url: https://charts.jetstack.io
```
Comment (michaeljguarino) [inline static/compatibilities/cert-manager.yaml:4]: this should probably be `helm_repository_url`

File: static/compatibilities/ingress-nginx.yaml [hunk 2]
```diff
@@ -1,15 +1,18 @@
 icon: https://github.com/pluralsh/plural-artifacts/blob/main/ingress-nginx/plural/icons/nginx.png?raw=true
 git_url: https://github.com/kubernetes/ingress-nginx
 release_url: https://github.com/kubernetes/ingress-nginx/releases/tag/controller-v{vsn}
+chart_url: https://helm.nginx.com/stable
```
Comment (michaeljguarino) [inline static/compatibilities/ingress-nginx.yaml:4]: this isn't the right helm repository, it's for `nginx` enterprise version of the ingress controller not `ingress-nginx` which is the kubernetes project maintained one.

Comment (seemywingz) [reply static/compatibilities/ingress-nginx.yaml]: aaah, right. I developed the chart updated using `cert-manager` and was focused on fixing the `ingress-nginx` scraper when I added the URL. it should be right now.

**Review by michaeljguarino — APPROVED** [review]
There are obviously a lot more of these w/ helm repos we should backfill but this is a good start

**Pull request merged**